src/QsCompiler/LanguageServer/Utils.cs

```diff
@@ -224,9 +224,9 @@ public override void Initialize(IEventSource eventSource)
 
     internal static class DotNetSdkHelper
     {
-        private static readonly Regex DotNet31Regex = new Regex(@"^3\.1\.\d+", RegexOptions.Multiline | RegexOptions.Compiled);
+        private static readonly Regex DotNet31Regex = new Regex(@"^6\.0\.\d+", RegexOptions.Multiline | RegexOptions.Compiled);
```

Comment:
You want to rename the variable too.

Reply:
Oh. Of course. Sorry for missing that. :)